modules/standards/test/standard-definition-authentication.test.ts

```diff
@@ -0,0 +1,185 @@
+import http from "http";
+import PhenylHttpServer from "@phenyl/http-server";
+import PhenylRestApi from "@phenyl/rest-api";
+import PhenylHttpClient from "@phenyl/http-client";
+import { GeneralTypeMap, KvsClient, Session } from "@phenyl/interfaces";
+import { createEntityClient } from "@phenyl/memory-db";
+import { StandardUserDefinition } from "../src/";
+import assert from "assert";
+
```

Comment:
for mocha, these methods need to be imported manually
```suggestion
import { after, before, describe, it } from "mocha";

```